_drafts/CocoaPods-1.9.0-beta.markdown

```diff
@@ -0,0 +1,142 @@
+
+```ruby
+Pod::Spec.new do |s|
+  s.name         = 'Networking'
+  s.version      = '1.0.0'
+
+  # ...rest of attributes here
+
+  s.test_spec 'Tests' do |test_spec|
+    test_spec.scheme = {
+        :code_coverage => true, :environment_variables => {'FOO' => 'BAR' }
+    }
+  end
+```
+
+## Swift Version Variants
+
+As new Swift versions are released, Pod authors have added support for compiling their source with multiple Swift versions by using the `swift_versions` Podspec DSL.
+
+To better support this, the Podfile DSL was updated in [1.7] to allow users to specify what versions of Swift their apps support, using the `supports_swift_version` DSL.
+
+If two different targets include the same dependency but require _different_ Swift versions, CocoaPods will now create multiple different targets for the same Pod to accomodate the different Swift versions used.
+
+Given the following Pod:
+
+```ruby
+Pod::Spec.new do |s|
+  s.name         = 'CannonPodder'
+  s.version      = '1.0.0'
+  # ...
+  s.swift_versions = ['4.0', '5.0']
+end
+```
+
+and the following Podfile:
+
+```ruby
+target 'SampleApp' do
+  supports_swift_version '< 5.0'
+  pod 'CannonPodder'
+end
+
+target 'OtherApp' do
+  supports_swift_version '>= 5.0'
+  pod 'CannonPodder'
+end
+```
+
+Two different versions of the `CannonPodder` target will be created.
+
+<center>
+{% breaking_image /assets/blog_img/CocoaPods-1.9.0/swift_variants.png %}
+</center>
+
+This ensures the correct Swift version is used for each target that depends on the Pod.
+
+## `use_frameworks!` Linkage Customization
+
+iOS 8.0 introduced the ability for apps to ship dynamically-linked frameworks. To support this, CocoaPods introduced the `use_frameworks!` DSL which made all Pods be compiled as dynamically-linked frameworks. Frameworks do not have any inherent _linkage_, but Swift initially required dynamic linking to be used in iOS apps. It is possible to wrap a statically-linked library in a `.framework ` bundle.
+
+Now that Swift supports static linking, CocoaPods has expanded this DSL to allow specifying the type of linkage preferred.
+
+```ruby
+use_frameworks! :linkage => :static
+```
+
+This is the first step in providing more control to CocoaPods users over how Pods are packaged and linked into their dependent binaries. For more details, check out the [RFC][use-frameworks-linkage-rfc].
+
+## What's Next
+
+In future versions, we will continue to expand the customization points for how pods are integrated into projects.
+
+Thanks to all of our contributors who helped make this release happen!
+
+Checkout the [changelog] to see the full list of changes.
+
+
+[1.7]: {% post_url 2019-02-22-CocoaPods-1.7.0-beta %}
+[use-frameworks-linkage-rfc]: https://github.com/CocoaPods/CocoaPods/issues/9099
+[changelog]: TODO REPLACE WITH CHANGELOG LINK
```

Comment:
TODO: add Github release link here

Reply:
you can already put the link of what would be 1.9.0, I've done the same in prior blog posts. Once the release goes out the link would start working.

Reply:
genius. thank you